.gitignore

```diff
@@ -10,3 +10,6 @@ docs/book/
 
 # Add this to your global .gitignore but whatever
 .DS_Store
+
+# Don't add test WASM blobs, CI generates them on their own
+*.wasm
```

Comment:
- I think we can remove this. 
- Contributors should know to not add these files to their commit(s)
- The `.DS_Store` technically shouldn't be there either (I added that when I was having a bad day I believe).